tutorials/learnsqlonline.org/en/Group by.md

```diff
@@ -1,3 +1,8 @@
+Introduction
+------------
+
+To discover more courses about data manipulation and SQL functions, register to DataCamp's [SQL](https://datacamp.pxf.io/DKLN45) tutorials. Learn Data Science by completing interactive coding challenges and watching videos by expert instructors. [Start Now!](https://datacamp.pxf.io/Kjxv6a)
```

Comment:
Hey,
I actually wanted you to change the default template somehow so we won't need to add this to each tutorial... sorry for not being clear on that